Lib/reprlib.py

```diff
@@ -60,15 +71,33 @@ def repr(self, x):
         return self.repr1(x, self.maxlevel)
 
     def repr1(self, x, level):
-        typename = type(x).__name__
+        _type = type(x)
```

Comment:
Style nit: underscored names are rarely used for local variables. Common name for the variable holding a type is `cls` or (rarer) `typ`, `tp`.